src/box/memtx_tx.c

```diff
@@ -1769,6 +1795,88 @@ memtx_tx_handle_point_hole_write(struct space *space, struct memtx_story *story,
 	mh_point_holes_del(ht, pos, 0);
 }
 
+static bool
+memtx_tx_tuple_matches(struct index *index, struct tuple *tuple,
```

Comment:
This needs the index just to extract a key def from it, let's consume it directly.